x-pack/test/apm_api_integration/common/apm_api_supertest.ts

```diff
@@ -60,3 +57,8 @@ Body: ${JSON.stringify(res.body)}`
     this.res = res;
   }
 }
+
+export interface SupertestReturnType<TEndpoint extends APIEndpoint> {
+  status: number;
+  body: NonNullable<APIReturnType<TEndpoint>>;
```

Comment:
Why is `NonNullable` necessary here?

Reply:
That's what we had in the tests already (moved that over to this util). But will remove it and see what breaks.